src/libs/modulegroups.c

```diff
@@ -1433,18 +1397,20 @@ static void _manage_editor_module_add_popup(GtkWidget *widget, gpointer data)
         g_object_set_data(G_OBJECT(bt), "group", gr);
         g_object_set_data(G_OBJECT(bt), "popup", pop);
         gtk_label_set_xalign(GTK_LABEL(gtk_bin_get_child(GTK_BIN(bt))), 1.0);
-        g_signal_connect(G_OBJECT(bt), "button-press-event", G_CALLBACK(_manage_editor_module_add), NULL);
+        g_signal_connect(G_OBJECT(bt), "button-press-event", callback, data);
 
         // does it belong to recommended modules ?
         GtkWidget *vbc = vb2;
         if(((module->default_group() & IOP_GROUP_BASIC) && g_strcmp0(gr->name, _("base")) == 0)
            || ((module->default_group() & IOP_GROUP_COLOR) && g_strcmp0(gr->name, _("color")) == 0)
            || ((module->default_group() & IOP_GROUP_CORRECT) && g_strcmp0(gr->name, _("correct")) == 0)
            || ((module->default_group() & IOP_GROUP_TONE) && g_strcmp0(gr->name, _("tone")) == 0)
-           || ((module->default_group() & IOP_GROUP_EFFECT) && g_strcmp0(gr->name, _("effect")) == 0)
+           || ((module->default_group() & IOP_GROUP_EFFECT)
+               && g_strcmp0(gr->name, C_("modulegroup", "effect")) == 0)
```

Comment:
Ah that was it ! Now the question is how could this have been working for you :( Some code are mystery to me :)

Reply:
because I mostly use dt in English...

Reply:
All clear then !